app/assets/javascripts/sessions.coffee

```diff
@@ -1,3 +1,10 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+ $ ->
```

Comment:
Cool..

Just did a quick check with code only in the sessions.coffee file and stuff
appears to work OK

On Thu, Apr 9, 2015 at 1:07 PM, Ben Woodall <notifications@github.com>
wrote:

> In app/assets/javascripts/sessions.coffee
> <https://github.com/benwoody/shrtnr/pull/4#discussion_r28097597>:
>
> > @@ -1,3 +1,10 @@
> >  # Place all the behaviors and hooks related to the matching controller here.
> >  # All this logic will automatically be available in application.js.
> >  # You can use CoffeeScript in this file: http://coffeescript.org/
> > +
> > + $ ->
>
> No need to create this method in each coffeescript file. I'd say keep one
> copy here in sessions.coffee since sessions are used everywhere. If you
> check the source, You'll see that there are now 7 of these functions in the
> code base.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benwoody/shrtnr/pull/4/files#r28097597>.
>



-- 
Joe Simpson
“Reasonable people adapt themselves to the world. Unreasonable people
attempt to adapt the world to themselves. All progress, therefore, depends
on unreasonable people.”
George Bernard Shaw